apps/lib/s_socket.c

```diff
@@ -315,6 +330,8 @@ int do_server(int *accept_sock, const char *host, const char *port,
 #ifdef AF_INET6
     if (sock_family == AF_INET6)
         sock_options |= BIO_SOCK_V6_ONLY;
+    if (tfo)
+        sock_options |= BIO_SOCK_TFO;
```

Comment:
Is this intentionally within the `ifdef AF_INET6` guard?

Reply:
What happens if this is a UDP socket?

Reply:
IPv6 Looks to have been a rebase error.
It should be a no-op for UDP and SCTP, as this sets a flag on the BIO and behavior changes on the connect (which doesn't happen with UDP).

Reply:
Fixed